docs/building-apps/wallet/sep10.mdx

```diff
@@ -130,6 +140,28 @@ const getAuthToken = async () => {
 };
 ```
 
+```dart
+// delegate client signing
+String jwtToken = await webAuth.jwtToken(
+  userAccountId,
+  [userKeyPair],
+  clientDomain: "demo-wallet-server.stellar.org",
+  clientDomainSigningDelegate: (transaction) async {
+    // delegate client signing e.g.:
+    return myWalletSigner.signTransaction(transaction, clientKeyPair);
```

Comment:
Is `clientDomainSigningDelegate` just a lambda?

Reply:
Hi @Ifropc, it is an optional argument that accepts a callback function. This is the definition of it:

```
Future<String> Function(String transactionXdr)? clientDomainSigningDelegate
```